kickstart/write-presentation.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Create kickstart presentation
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Kickstart
+
+description: >-
+    Kickstart events require a presentation to go through what student robotics
+    is, what the competition cycle looks like, and details of the game.
+
+    The presentation needs to be given at each venue, so will be presented by
+    multiple people.
+
+    The presentation usually covers:
+
+    1. What student robotics is
+    2. Who the volunteers are
+    3. Schedule for the year
+        - How long the competition is
+        - Tech days date, time, etc..
+    4. General robot design tips
+    5. The kit
+        - Hardware
+        - Software
+        - The docs
+        - The forums
+    6. Health and safety
+        - How to charge your batteries
+    7. Awards
+    8. Schedule for the rest of kickstart
```

Comment:
Unfortunately the formatting of this doesn't survive loading quite as desired. I suggest either using a slightly different style of yaml text block or changing the indent so this is preserved.

You can test to see what the loaded content looks like by running `scripts/import.py kickstart/write-presentation 2020`. currently for this task that shows:
```
#2675: Create kickstart presentation
  Kickstart events require a presentation to go through what student robotics is, what the competition cycle looks like, and details of the game.
  The presentation needs to be given at each venue, so will be presented by multiple people.
  The presentation usually covers:
  1. What student robotics is 2. Who the volunteers are 3. Schedule for the year
      - How long the competition is
      - Tech days date, time, etc..
  4. General robot design tips 5. The kit
      - Hardware
      - Software
      - The docs
      - The forums
  6. Health and safety
      - How to charge your batteries
  7. Awards 8. Schedule for the rest of kickstart

  Original: [recurring-task:kickstart/write-presentation]

  Dependencies:

   * #2674 Write the rules for the SR2020 game
```

Reply:
Fixed with a single character! https://github.com/srobo/recurring-tasks/pull/11/commits/e8326d3c09b0e3612f59d4ea5c5e8e0686789153

Reply:
Just to check - have you tried also importing the task into GitHub?